hypha/apply/funds/views.py

```diff
@@ -1157,3 +1162,33 @@ def get_submission_values(self):
                 values.append(value)
 
         return {'total': sum(values), 'average': round(mean(values))}
+
+
+@method_decorator(login_required, name='dispatch')
+class ReviewLeaderboard(SingleTableMixin, FilterView):
+    filterset_class = LeaderboardFilter
+    table_class = LeaderboardTable
+    table_pagination = False
+    template_name = 'funds/review_leaderboard.html'
+
+    def dispatch(self, request, *args, **kwargs):
+        is_staff = request.user.is_apply_staff
+        is_reviewer = request.user.is_reviewer
+
+        if not (is_staff or is_reviewer):
```

Comment:
We can use UserPassesTestMixin as used in other [areas](https://github.com/OpenTechFund/hypha/blob/ceb74015590ae24d1412478505faeda6951c080d/hypha/apply/review/views.py#L61) of the codebase.

Reply:
Fixed, thanks.